frontend/src/pages/login.jsx

```diff
@@ -27,44 +29,72 @@ const Login = () => {
 
   return (
     <>
-      <main>
-        <section>
-          <div>
+      <main className="bg-customCream h-screen flex items-center justify-center">
+        {/* Logo in the top left corner with link back to homepage */}
+        <Link
+          to="/"
+          className="text-customBlue text-4xl font-bold font-inknut absolute top-4 left-6"
+        >
+          SKINrecs
+        </Link>
+        <div className="w-full max-w-md">
+          {/* Log in prompt */}
+          <div className="bg-customBlue shadow-lg rounded-lg p-8">
+            <h2 className="text-center text-customCream text-3xl font-bold mb-6 font-inknut">
+              Login
+            </h2>
             <form>
-              <div>
-                <label htmlFor="email-address">Email address</label>
+              {/* Input email address */}
+              <div className="mb-4">
+                <label htmlFor="email-address" className="sr-only">
+                  Email address
+                </label>
                 <input
                   id="email-address"
                   name="email"
                   type="email"
                   required
-                  placeholder="Email address"
+                  className="w-full p-3 rounded border border-gray-300"
+                  placeholder="Email Address"
                   onChange={(e) => setEmail(e.target.value)}
                 />
               </div>
 
-              <div>
-                <label htmlFor="password">Password</label>
+              {/* Input password */}
+              <div className="mb-6">
+                <label htmlFor="password" className="sr-only">
+                  Password
+                </label>
                 <input
                   id="password"
                   name="password"
                   type="password"
                   required
+                  className="w-full p-3 rounded border border-gray-300"
```

Comment:
Same thing here I'm not sure if the text is supposed to be black. I think the searchbar uses customGray